docker/polkastats-backend/sql/polkastats.sql

```diff
@@ -40,5 +40,16 @@ CREATE TABLE event (
    data TEXT NOT NULL,
    PRIMARY KEY ( block_number, event_index ) 
 );
+
+CREATE TABLE rewards_info (
+   block_number BIGINT NOT NULL,
+   session_index INT NOT NULL,
+   stash_id VARCHAR(50),
+   commission BIGINT,
+   era_rewards TEXT,
+   stake_info TEXT,
+   timestamp INT NOT NULL,
+   PRIMARY KEY ( block_number, session_index )  
```

Comment:
To avoid the duplicate key issue replace this:

```
PRIMARY KEY ( block_number, session_index )
```

by:

```
PRIMARY KEY ( block_number, session_index, stash_id )
```

Reply:
really easy fix xD Should have seen that. Thank you Mario! 😄